jujugui/static/gui/src/app/components/overview-action/overview-action.js

```diff
@@ -0,0 +1,79 @@
+/*
+This file is part of the Juju GUI, which lets users view and manage Juju
+environments within a graphical interface (https://launchpad.net/juju-gui).
+Copyright (C) 2015 Canonical Ltd.
+
+This program is free software: you can redistribute it and/or modify it under
+the terms of the GNU Affero General Public License version 3, as published by
+the Free Software Foundation.
+
+This program is distributed in the hope that it will be useful, but WITHOUT
+ANY WARRANTY; without even the implied warranties of MERCHANTABILITY,
+SATISFACTORY QUALITY, or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU Affero
+General Public License for more details.
+
+You should have received a copy of the GNU Affero General Public License along
+with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+'use strict';
+
+YUI.add('overview-action', function() {
+
+  juju.components.OverviewAction = React.createClass({
+
+    /**
+      Returns the supplied classes with the 'hidden' class applied if the
+      value is falsey.
+      @method _valueClasses
+      @returns {String} The collection of class names.
+    */
+    _valueClasses: function() {
+      return classNames(
+        'overview-action__value',
```

Comment:
Seems we are reusing the component name `overview-action` a number of times in this class. Would it be better to create a variable if it?